client/src/components/songDetails/songDetails.css

```diff
@@ -0,0 +1,48 @@
+@font-face {
+  font-family: "ABeeZee";
+  src: url("../../assets/fonts/ABeeZee-Regular.ttf") format("truetype");
+}
+
+* {
```

Comment:
move this to main.css/app.css. this is relevant to all the application

Reply:
do you mean to put it in my CSS file and **also** in the App.css file? or just in the App.css file in the root element?